backend/src/routes/api/medication.routes.js

```diff
@@ -0,0 +1,17 @@
+import Medication, { medicationValidator } from '../../models/Medication.js';
+import { StatusCodes } from 'http-status-codes';
+
+const MedicationRoutes = (router) => {
+  router.delete('me/meds/:id', deleteMedication);
+};
+
+export async function deleteMedication(req, res, next) {
+  try {
+    const medication = await Medication.findByIdAndDelete(req.params.id);
```

Comment:
if a document is found and removed this method will return an object with the document else it will return null
Let's throw an error in the latter scenario so we can inform that no record was found